app/Http/Controllers/Equipment/RepairsController.php

```diff
@@ -153,4 +166,27 @@ private function updateBreakageStatus(Breakage $breakage, Request $request)
         ]);
         Notify::success('Breakage updated');
     }
+
+    /**
+     * Stream an image.
+     * 
+     * @param $id
+     * @param $image
+     * 
+     * @return mixed
+     */
+    public function streamImage($id, $imageId) {
+        $breakage = Breakage::findOrFail($id);
+        $this->authorize('view', $breakage);
+
+        $image = BreakageImage::findOrFail($imageId);
```

Comment:
This is a very minor comment, but I'd still recommend using something like `$breakage->images()->where('id', $imageId)->firstOrFail()` - it seems overkill but it makes sure that someone doesn't try to fetch an image for a different breakage.

It's not particularly problematic in this instance, but if we restricted who could access specific breakages this would allow someone to bypass that authorisation check for the images :slightly_smiling_face:

Election manifestos is an example where we do something similar: https://github.com/backstage-technical-services/website/blob/main/app/Http/Controllers/Elections/NominationController.php#L82